src/index.spec.jsx

```diff
@@ -50,3 +50,12 @@ it('allows to define a custom component', () => {
 
   expect(toJson(wrapper)).toMatchSnapshot();
 });
+
+it.only('applies aria attributes to <object> to avoid screenreader issues', () => {
```

Comment:
Sorry just realised that `.only` was still in here, i'll send a follow up PR since this has already been merged